### ISSUE TITLE
after update to ver. 1.0 only blank screen 

### ISSUE THREAD

**Issue by CHazz** [e-alfred/flowupload#99]
hi after update to 1.0.0 on nextcloud 18 have only blank screen with new destination and not working settings ... any tip for fix up ?

not see any errors in log 
![flowupload 1 0](https://user-images.githubusercontent.com/106156/83210879-218a5b00-a15c-11ea-80b2-4ad670b1a630.jpg)

**Comment by e-alfred** [comment]
You have to select "new destination" on the left pane to select the folder where you want to upload your files to. You can select multiple folders now and they will be saved into the database of your Nextcloud installation. The settings are empty for now, but will be filled later on with options (especially #50 and #75).

@TessyPowder Maybe we should put some information into the right pane to tell users how they  have to use the app.

**Comment by CHazz** [comment]
hi, i ofcourse select destination ... but still nothing work :(

**Comment by e-alfred** [comment]
Did you empty your browser cache? Can you open the browser console to see if any error is shown there?

**Comment by CHazz** [comment]
yes, this error is in all browsers and have all  my users

**Comment by CHazz** [comment]
i get this error
jquery.js:8630 GET https://cloud.mydomain.xxx/apps/flowupload/directories 400

**Comment by e-alfred** [comment]
Can you tell us a bit more about your environment? Do you have your data on external storage?

**Comment by CHazz** [comment]
data is on local storage
debian 10, MariaDB 10.4, Nginx 1.19+security mod, PHP7.4-FPM, Redis 6.0.4 for local

**Comment by TessyPowder** [comment]
It seems like the database migration didn't work. Can you try to remove flowupload and reinstall it ?

**Comment by CHazz** [comment]
yes .. maybe 3x i try reinstall complete on backup system and same problem

**Comment by e-alfred** [comment]
> Nginx 1.19+security mod

As it is returning a 400 bad request it might have to do with any additional security modules (mod_evasive with Apache is somewhat problematic), can you test it without this module? Does it log any error in the NGINX log or any other security log?

**Comment by CHazz** [comment]
i not see any new errors :) but i try on pure stable nginx now

**Comment by TessyPowder** [comment]
Can you browse the nextcloud databse (with phpmyadmin or such) and find out if the table oc_flowupload_directories exists ? It should automatically created at the install, but according to the log it doesn't exist.

If it exists try removing this app, removing the oc_flowupload_directories table manually and reinstalling flowupload.

**Comment by TessyPowder** [comment]
I really hope this is no problem in the DB Migration file i wrote 😬

**Comment by TessyPowder** [comment]
On my server it worked and my setup is relatively similar to yours. So i am not sure what could go wrong

**Comment by CHazz** [comment]
DB really not exist .. but reinstall dont create DB .. i check source of flow to creator DB manualy then

**Comment by TessyPowder** [comment]
Thats weird. The Structure can be seen in the code here:
![Screenshot from 2020-06-03 16-05-56](https://user-images.githubusercontent.com/28999431/83646632-34fe5180-a5b4-11ea-8c4a-10cb4f077035.png)

**Comment by TessyPowder** [comment]
i will try if this will happen on a fresh 18.0.4 server.
I only use docker, but i would be extremely surprised if that would make a difference.
I i cannot reproduce this error something has to be messed up with your instance / database @CHazz

**Comment by TessyPowder** [comment]
nobody else in our [call for testing](https://help.nextcloud.com/t/call-for-testing-flowupload-1-0/81557) for the current release on the forum had this problem, so i guess you are the only one facing this problem. Could it be a permissions issue with you database ?

**Comment by TessyPowder** [comment]
Could you post if any errors occur at the app install ?

**Comment by CHazz** [comment]
i try create table manualy..now no have any one error .. in nextcloud, php, nginx .. nothing .. after reinstal nothing .. and still not working :/ old version working perfect, but after update to 1.0.0 this :(

**Comment by CHazz** [comment]
hi, i try complete reintall with database ,, and have new error .. maybe this help us


[no app in context] Error: Symfony\Component\Routing\Exception\RouteNotFoundException: Unable to generate a URL for the named route "flowupload.page.index" as such route does not exist. at <<closure>>

 0. /var/www/html/lib/private/Route/Router.php line 339
    Symfony\Component\Routing\Generator\UrlGenerator->generate("flowupload.page.index", [], 1)
 1. /var/www/html/lib/private/Route/CachingRouter.php line 59
    OC\Route\Router->generate("flowupload.page.index", [], false)
 2. /var/www/html/lib/private/URLGenerator.php line 82
    OC\Route\CachingRouter->generate("flowupload.page.index", [])
 3. /var/www/html/lib/private/NavigationManager.php line 293
    OC\URLGenerator->linkToRoute("flowupload.page.index")
 4. /var/www/html/lib/private/NavigationManager.php line 114
    OC\NavigationManager->init()
 5. /var/www/html/lib/private/TemplateLayout.php line 87
    OC\NavigationManager->getAll()
 6. /var/www/html/lib/private/legacy/template.php line 184
    OC\TemplateLayout->__construct("user", "settings")
 7. /var/www/html/lib/public/AppFramework/Http/TemplateResponse.php line 167
    OC_Template->fetchPage({serverData: {ap ... }})
 8. /var/www/html/lib/private/AppFramework/Http/Dispatcher.php line 123
    OCP\AppFramework\Http\TemplateResponse->render()
 9. /var/www/html/lib/private/AppFramework/App.php line 125
    OC\AppFramework\Http\Dispatcher->dispatch(OCA\Settings\Con ... {}, "viewApps")
10. /var/www/html/lib/private/AppFramework/Routing/RouteActionHandler.php line 47
    OC\AppFramework\App::main("OCA\\Settings\\ ... r", "viewApps", OC\AppFramework\ ... {}, {category: "",id ... "})
11. <<closure>>
    OC\AppFramework\Routing\RouteActionHandler->__invoke({category: "",id ... "})
12. /var/www/html/lib/private/Route/Router.php line 299
    call_user_func(OC\AppFramework\ ... {}, {category: "",id ... "})
13. /var/www/html/lib/base.php line 1008
    OC\Route\Router->match("/settings/apps")
14. /var/www/html/index.php line 38
    OC::handleRequest()

GET /settings/apps

**Comment by krakazyabra** [comment]
Hi! I have the same problem. We're using S3 as primary storage. When I upload a folder, it appears, but there are no files inside. Console is fine, there are no errors.
